_topic_maps/_topic_map.yml

```
@@ -1261,6 +1261,12 @@ Topics:
   - Name: Disaster recovery considerations
     File: nbde-disaster-recovery-considerations
     Distros: openshift-enterprise,openshift-origin
+- Name: zero trust workload identity manager for Red Hat OpenShift
+  Dir: zero_trust_workload_identity_manager
+  Distros: openshift-enterprise
+  Topics:
+  - Name: zero-trust workload identity manager for Red Hat OpenShift overview
```

An alternative name could be:

But I do not have a preference. I'm only calling it out because NBDE above it uses "About. . . " in the first section. No preference though.

```suggestion
  - Name: About zero trust workload identity manager for Red Hat OpenShift
```

---

I think I'm going to keep it the way it is now. zero trust documentation will closely follow the cert-manager documentation. This is the way it is in cert-manager so I think I'll keep it this way for now. Thanks.